indico/modules/events/papers/client/js/components/SubmitRevision.jsx

```diff
@@ -74,15 +41,14 @@ export default function SubmitRevision() {
           <Translate>Upload corrected revision</Translate>
         </div>
         <div className="i-box-content">
-          <FileSubmission onChange={newFiles => setFiles(newFiles)} disabled={submitting} />
-          <Button
-            onClick={submitFiles}
-            content={Translate.string('Submit new revision')}
-            style={{marginTop: 10}}
-            disabled={files.length === 0 || submitting}
-            loading={submitting}
-            primary
-          />
+          <RevisionSubmissionForm eventId={eventId} contributionId={contributionId}>
+            <Button
```

Comment:
This should probably use a `FinalSubmitButton` which gets automatically disabled while submitting etc.